docs/scripts/update-all-docs.sh

```diff
@@ -111,3 +111,15 @@ echo ".. _list_toolchains:" > $VERSION_SPECIFIC_DIR/toolchains.rst
 echo "*(see also* \`\`eb --list-toolchains\`\` *)*" >> $VERSION_SPECIFIC_DIR/toolchains.rst
 echo >> $VERSION_SPECIFIC_DIR/toolchains.rst
 eb --list-toolchains --output-format rst >> $VERSION_SPECIFIC_DIR/toolchains.rst
+
+#  available toolchain options
+echo ".. _avail_toolchain_opts:" > $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo "*(see also* \`\`eb --avail-toolchain-opts <tcname>\`\` *)*" >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+IFS=$'\n'
+for i in `eb --list-toolchains | sed 1d`
+do
+    tc=`echo $i | cut -d ':' -f1`
+    eb --avail-toolchain-opts ${tc//[[:blank:]]/} --output-format rst >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
```

Comment:
ok, this works now (cfr. http://boegel-eb.readthedocs.io/en/test/version-specific/toolchain_opts.html), but two requests:

* sort toolchains alphabtically
* can we add quicklinks for each of the toolchains on the top of the page? that'll involve inserting labels for each toolchain...